deploy/examples/create-external-cluster-resources.py

```diff
@@ -998,7 +998,7 @@ def get_rbd_node_caps_and_entity(self):
         return caps, entity
 
     def get_healthchecker_caps_and_entity(self):
```

Comment:
```suggestion
    def get_defaultUser_caps_and_entity(self):
```
Let's change the function name also